chrome-extension/manifest.json

```diff
@@ -12,6 +12,7 @@
 		"scripting"
 	],
 	"host_permissions": [
+		"file:///*",
```

Comment:
It seems that "optional_host_permissions" is also fine.

[Restricting use of file: URLs in tabs and windows APIs](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/ZtCvVISQU54/m/Qv_wCnnUAQAJ)

> I ended up adding "optional_host_permissions": [ "file:///*" ]
> This way the "Allow access to file URLs" is shown, navigation works, but I don't actually get read permission.